src/QueryHandle.cpp

```diff
@@ -210,7 +210,7 @@ auto QueryHandle::connect() -> bool
             m_connection.GetPassword().c_str(),
             m_connection.GetDatabase().c_str(),
             m_connection.GetPort(),
-            "0",
+            m_connection.GetSocket().c_str(),
```

Comment:
How does the mysql_real_connect() prioritize host/port and the socket?

Reply:
If the host is NULL or localhost, it uses the socket. If it's a different value (e.g. "db.rekt.lod"), it uses the port. So, changing the env from "localhost" (the current default) to something else "db.rekt.lod" will cause it to ignore the unix-socket and use the port.